### DISCUSSION TITLE
Unable to create digital ocean account

### DISCUSSION THREAD

**Discussion by alirazab22** [community/community#45966] [GitHub Education]
Hello there,

I uploaded my student ID and got benefits from github but unfortunately, whenever I try to register digital ocean account via this github account, it shows this account doesn't exist. Please help me to resolve this issue.
Thank you

**Comment by doguedogue** [comment]
You have to enter here
https://cloud.digitalocean.com/

and sign in with your GitHub 
![image](https://user-images.githubusercontent.com/23409026/215914928-d1e83d3c-0d49-4381-ac7f-31c19f1e92b6.png)

**Reply by alirazab22** [reply]
Hello there,

Thank you for responding but unfortunately, I'm trying the same method you described, when I click on "Sign in with Github" it shows "No account is exist with this email address" please tell me what to do? I really digital ocean account.

**Reply by l-smh** [reply]
hello , Did you find a solution to the problem because I suffer from it too؟

**Reply by kokoshneta** [reply]
@l-smh Did you try the method in [my reply below](#discussioncomment-5184380)?

**Comment by alirazab22** [comment]
Hello Sir,

This is what I've already mentioned that when I try to signup, it shows no account is exist. I logged in my github account in which I got approval for student pack, I tried to register with two methods, first one is, like you mentioned, sign in github account and got to https://cloud.digitalocean.com/ and then click on "Sign in with Github" but when I click on it, it shows "An account with this email address was not found. Please try again or contact support<https://www.digitalocean.com/support/contact/> if you are unable to access your account.".

Then I went into the github and click on student benefits where I also got link to register digital ocean it also redirects me to cloud.digitalocean.com and then again when I tried to register with "Sign in with Gitub" it shows same error.

Please help me
[https://www.digitalocean.com/assets/media/logo-a721c4a7.png]<https://cloud.digitalocean.com/>
DigitalOcean – The developer cloud<https://cloud.digitalocean.com/>
Helping millions of developers easily build, test, manage, and scale applications of any size – faster than ever before.
cloud.digitalocean.com

________________________________

**Comment by kokoshneta** [comment]
Had the same issue here.

First time I clicked on “Sign In with GitHub”, it asked for permission to connect DigitalOcean with my GitHub account, which I allowed. DigitalOcean shows up as active in my GitHub connections as well now. But after connecting – and on subsequent attempts, immediately after clicking “Sign In with GitHub” – I got the exact same message as @alirazab22:

> An account with this email address was not found. Please try again or [contact support](https://www.digitalocean.com/support/contact/) if you are unable to access your account.

About to give up, I instead clicked on [“Sign Up”](https://cloud.digitalocean.com/registrations/new) and only _then_ discovered that there’s an additional “Sign Up with GitHub” option. Unlike most sites with a GitHub login option, you can’t just log in directly – you have to _sign up through GitHub_ first and add a payment method, and _then_ you can log in.

**Comment by surajrimal07** [comment]
same issue here too, any luck so far?

**Comment by Ayshyama** [comment]
Have the same issue. I registered using GitHub Student Pack.
It was working well for around 5 months because I stayed logged in just in 1 browser on 1 machine. But after I reinstalled my OS and tried to log in again using the same GitHub account I got the error 'An account with this GitHub sign-in was not found.'
